src/script/components/InputBar/InputBar.tsx

```diff
@@ -565,13 +566,33 @@ const InputBar = ({
 
   const onGifClick = () => openGiphy(inputValue);
 
+  const ping = () => {
+    void messageRepository.sendPing(conversationEntity).then(() => {
+      window.setTimeout(() => setIsPingDisabled(false), CONFIG.PING_TIMEOUT);
+    });
+  };
+
+  const totalConversationUsers = conversationEntity.participating_user_ets().length;
+
   const onPingClick = () => {
     if (conversationEntity && !pingDisabled) {
-      setIsPingDisabled(true);
-
-      messageRepository.sendPing(conversationEntity).then(() => {
-        window.setTimeout(() => setIsPingDisabled(false), CONFIG.PING_TIMEOUT);
-      });
+      if (conversationEntity.is1to1() || totalConversationUsers < 3) {
```

Comment:
What about having a constant for this `"3"`? e.g. `MAX_USERS_TO_PING_WITHOUT_ALERT`?